spec/calculator_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'calculator'
+
+describe Calculator do
+  subject(:calculator) { described_class.new(order, takeaway) }
+  let(:takeaway) { double :takeaway}
+  let(:menu) { {ribs: 3,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.